README.md

```diff
@@ -48,7 +48,15 @@ You can download the source with the command:
 
 For those new to Rails, the following tutorial will help get you up to speed with configuring a [Rails environment](http://guides.rubyonrails.org/getting_started.html).
 
-When ready, run `script/setup`. If the script succeeds you're ready to start developing. If not, take a look at the output as it should be informative enough to help you troubleshoot.
+First, you need to create the database user the app will use by manually typing the following in your terminal:
+
+```sh
+$ sudo -u postgres createuser ofn --superuser --createdb --pwprompt
+```
+
+This will create the "ofn" user as superuser and allowing it to create databases. Note the command will prompt for the password twice. Use "food".
```

Comment:
The password is `f00d`, not `food`. We replace the o's with zeros to make it extra secure :wink: :wink:

Reply:
:see_no_evil: yep, typo